server/routes/incidents_sse.py

```
@@ -1,32 +1,32 @@
-        try:
-            message_queue.put_nowait(incident_data)
-        except queue.Full:
-            logger.warning("Message queue full for scope %s, dropping message", scope_key)
-        except Exception as e:
-            logger.error("Error sending message to queue for scope %s: %s", scope_key, e)
+    channel = f"{_CHANNEL_PREFIX}{scope_key}"
+    try:
+        r = redis.from_url(os.getenv("REDIS_URL", "redis://redis:6379/0"), **get_redis_ssl_kwargs())
+        r.publish(channel, json.dumps(incident_data))
+        r.close()
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)